recipes/plasmapy/meta.yaml

```diff
@@ -19,13 +19,13 @@ requirements:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
   host:
-    - python >= 3.6
-    - cython >= 0.28
+    - python
```

Comment:
PlasmaPy requires 3.6 as we have fstrings basically everywhere. Could that potentially become a problem?

Reply:
You should only use this method of constraining the version for noarch packages. As this doesn’t have noarch: python, the constraint comes from the skip: True [py<36].

Also, version constraints should be formatted with one space before the operator and none after.

Reply:
Thanks, I'm learning a lot here! :)